provisioning_client/tests/hsm_client_tpm_ut/hsm_client_tpm_ut.c

```diff
@@ -127,6 +127,72 @@ static void on_umock_c_error(UMOCK_C_ERROR_CODE error_code)
 
 static TEST_MUTEX_HANDLE g_testByTest;
 
+char* umock_stringify_TPMA_SESSION(const TPMA_SESSION* value)
```

Comment:
Same comments as in for AMQP disposition.

Reply:
fixed mirrored AMQP disposition